### DISCUSSION TITLE
Structured report of compliance failures

### DISCUSSION THREAD

**Discussion by umarpro18** [diffplug/spotless#2027] [Q&A]
Hi Ned, Any updates on the below issues
https://github.com/diffplug/spotless/issues/811
https://github.com/diffplug/spotless/issues/655

I would like to capture the console result in a readable format to use it in our pipelines.
Example: ktlint produces a similar report "lint-results-ktlint.xml" from the hood.
args "--reporter=plain", "--reporter=plain?color=false,output=${buildDir}/reports/lint-results-ktlint.txt", "--color", "--reporter=checkstyle,output=${buildDir}/reports/lint-results-ktlint.xml", "**/src/**/*.kt"

**Comment by nedtwigg** [comment]
I expect this to happen by the summer or so.
